source/tools-and-extensions/archiving/HDB++.rst

```diff
@@ -165,64 +184,48 @@ HdbViewer
-partitioning period.
-
-.. note::
-    The C++ extraction library currently supports only the MySQL back-end.
-
-The *HdbExtractor++ multithread library* allows fetching the data from the legacy HDB and the new HDB++ MySQL schema in a simple Object Oriented way. An additional module provides a Qt interface to the HdbExtractor++ and a dedicated GUI, exploiting the MathGL framework, aimed
-at displaying mono and bidimensional data over time.
+Two libraries have been developed to the *historical data extraction*:
 
+* A java implementation, *libhdbpp-extraction-java*, has been used for the *HdbViewer* GUI and is a native choice for Java device servers. The HdbViewer Java framework, supports both legacy HDB archiving system, and the new HDB++ design.
+* A C++ implementation, dedicated *Qt/Qtango* based GUIs or to C++ Tango device servers.
+  The *HdbExtractor++ multithread library* allows fetching the data from the legacy HDB and the new HDB++ MySQL schema in a simple Object Oriented way. An additional module provides a Qt interface to the HdbExtractor++ and a dedicated GUI, exploiting the MathGL framework, aimed at displaying mono and bidimensional data over time.
 
-Some shared libraries provide the methods for writing to the database back-end.
-These libraries, written in C++, are addressed to the EventSubscriber Tango device server and
-their main purpose is to provide *an abstraction layer*. Actually, some shared objects are available
-implementing the abstraction layer and the specific interface:
-
-|libs|
-
-Figure 4: HDB++ Device Servers design
-
+ .. note::
+    The C++ extraction library currently supports only the MySQL back-end.
 
-* **libhdb++**: database abstraction layer, decouples the interface to the database back-end from the implementation.
-* **libhdbmysql**: legacy HDB schema support for MySQL back-end
-* **libhdb++mysql**: HDB++ schema support for MySQL back-end
-* **libhdb++cassandra**: methods from libhdb++ for Cassandra back-end
 
-These libraries allow reusing the EventSubscriber, the ConfigurationManager and the GUIs without changes.
 
 
 Source code
 -----------
 
 The source code is available on GitHub in the following repositories:
 
+.. _hdbpp-timescale-project: https://github.com/tango-controls-hdbpp/hdbpp-timescale-project
 .. _hdbpp-cm: https://github.com/tango-controls-hdbpp/hdbpp-cm
 .. _hdbpp-es: https://github.com/tango-controls-hdbpp/hdbpp-es
 .. _hdbpp-cm-es: https://github.com/tango-controls-hdbpp/hdbpp-cm-es
 .. _libhdbpp: https://github.com/tango-controls-hdbpp/libhdbpp
 .. _libhdbpp-mysql: https://github.com/tango-controls-hdbpp/libhdbpp-mysql
 .. _libhdbpp-mysql-legacy: https://github.com/tango-controls-hdbpp/libhdbpp-mysql-legacy
 .. _libhdbpp-cassandra: https://github.com/tango-controls-hdbpp/libhdbpp-cassandra
+.. _libhdbpp-timescale: https://github.com/tango-controls-hdbpp/libhdbpp-timescale
 .. _CassandraMonitor: https://github.com/tango-controls-hdbpp/CassandraMonitor
```

Comment:
I would add link to libhdbpp-postgresql and libhdbpp-elk

Reply:
It should be ok, now, or did I miss some other projects ?

Reply:
It's WIP, but hdbpp-benchmark could be mentioned as "a project to compare performances of different HDB++ backends using dockers"

Reply:
Added.